adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -3046,4 +3188,50 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
       loadParquetFragments(pathName, optPredicate = optPredicate, optProjection = optProjection)
     }
   }
+
+  /**
+   * Test if Parquet files are partitioned
+   *
+   * @param filePath Path in which to look for partitioned flag.
+   * @return Return True if partitioned flag found, False otherwise.
+   */
+
+  def checkPartitionedParquetFlag(filePath: String): Boolean = {
```

Comment:
+1

Reply:
At the moment application code (Mango) needs to itself use `checkPartitionedParquetFlag` to check to see if the data is partitioned to decide which load function (partitioned or not) to call from ADAM.  Thus `checkPartitionedParquetFlag` needs to not be private as it needs to be used outside ADAM.

This is because, due to parameter differences (no optPredicate and not opProjection) in  the partitioned parquet load functions), we don't have a way that I see to create a generic load function at in ADAM  that is agnostic to the "hive partitioned" status.

Reply:
Ah right, this conversation is going in circles a bit.  Could this method be renamed `isPartitioned` or something similar, starting with `is`?

Reply:
sure thing